### ISSUE TITLE
error: undefined

### ISSUE THREAD

**Issue by grzcyg** [ThemeFuse/Unyson#1700]
Hello, 
after update Unyson from 2.5.6 to 2.5.7 I've got this error "error:undefined" when I want to use a Text Block. 

How can I solve it?

**Comment by moldcraft** [comment]
Please give a screenshot.

**Comment by grzcyg** [comment]
![image](https://cloud.githubusercontent.com/assets/11816388/16538670/9ad330c2-402b-11e6-90a3-aebaaa6a93f4.png)

**Comment by moldcraft** [comment]
Please open browser console (F12), go to Network tab, open the Text Block shortcode and inspect the server response

![staticshot_03-07-2016_19-57-23](https://cloud.githubusercontent.com/assets/1475489/16546610/7e7c11a4-4158-11e6-9831-c6af51fa7767.png)

**Comment by grzcyg** [comment]
![image](https://cloud.githubusercontent.com/assets/11816388/16547237/be4a9a04-4166-11e6-823d-3acb75895935.png)


![image](https://cloud.githubusercontent.com/assets/11816388/16547247/f2ebe812-4166-11e6-8c98-242ab5202160.png)

**Comment by moldcraft** [comment]
Please enable a default WP theme Twenty*, and check if Text Block shortcode works

**Comment by moldcraft** [comment]
1. Which theme are you using?
2. Can you send me admin credentials for your site to `moldcraft[at]email.com`?

**Comment by moldcraft** [comment]
cc @andreiglingeanu

**Comment by andreiglingeanu** [comment]
@grzcyg @moldcraft I'd also want to take a look at the admin panel. Please ping me at `andrei.glingeanu[at]gmail.com`. Trying to reproduce this.

**Comment by andreiglingeanu** [comment]
Please test.

**Comment by bensigel** [comment]
Hello !

I have same problem with Wordpress 4.5.3 version with Unyson 2.5.7
When I click Text Block I have got error:undefined.

admin-ajax.php ( with internal error) respond:
<div class="fw-backend-option fw-backend-option-design-default fw-backend-option-type-wp-editor fw-row" id="fw-backend-option-fw-edit-options-modal-text">
			<div class="fw-backend-option-label fw-col-xs-12 fw-col-sm-3 fw-col-lg-2">
			<div class="fw-inner">
				<label for="fw-edit-options-modal-text">Content</label>
								<div class="fw-clear"></div>
			</div>
		</div>
		<div class="fw-backend-option-input fw-backend-option-input-type-wp-editor fw-col-xs-12 fw-col-sm-9 fw-col-lg-10 width-type-auto">
		<div class="fw-inner fw-pull-left">
						<div class="fw-inner-option">

**Comment by bensigel** [comment]
And every other elemets work well, but Text Block...

**Comment by andreiglingeanu** [comment]
Have you guys updated to the last master Unyson? I'm pretty sure it works now?

**Comment by bensigel** [comment]
In the wordpress menu I can't see any warm to update unyson...
How can I update ? Upload with ftp ?

**Comment by bensigel** [comment]
I've updated with ftp, but I got same error. Maybe http://themeforest.net/item/evoke-wordpress-theme/11827132 theme problem ?

**Comment by grzcyg** [comment]
@moldcraft @andreiglingeanu please, check email

**Comment by bensigel** [comment]
I've downgrade Unyson 2.5.6 it's perfect... This is my solution...

**Comment by andreiglingeanu** [comment]
@bensigel In the next release it will be fixed.

**Comment by grzcyg** [comment]
How can I downgrade to 2.5.6?

**Comment by bensigel** [comment]
Delete everything from wp-content/plugins/unyson. (unyson directory too)
And upload this https://github.com/ThemeFuse/Unyson/archive/v2.5.6.zip. And
I went to Wordpress Control Panel Unyson menu to download required modules
(Page Builder and Demo Content and Backup) and done.
But you can't see anything on website during this process...

2016-07-05 16:47 GMT+02:00 grzcyg <notifications@github.com>:

**Comment by bensigel** [comment]
Now working the Theme Settings menu too...

**Comment by grzcyg** [comment]
Thanks man, it's work

**Comment by bensigel** [comment]
Great !

**Comment by polterbyte** [comment]
First of all, sorry for opening ticket #1725, when this is clearly the same issue.

Second, if I understand the matter correctly, the only available solution at this point is downgrading Unyson to version 2.5.6, as instructed by @bensigel, above, correct?

The response I get for the admin-ajax.php request is:

{"wp-refresh-post-lock":{"new_lock":"1467792885:1"},"wp-auth-check":true,"server_time":1467792885}

Thanks.

**Comment by moldcraft** [comment]
https://github.com/ThemeFuse/Unyson-Extensions-Approval/issues/232

**Comment by moldcraft** [comment]
Add in theme `settings.php`

```php
$options = array(
	'p' => array(
		'type' => 'addable-popup',
		'popup-options' => array(
			'w' => array('type' => 'wp-editor'),
		),
	),
);
```

![screenshot from 2016-07-08 17-56-07](https://cloud.githubusercontent.com/assets/1475489/16691396/453e8742-4535-11e6-84e9-ab14253789ba.png)

**Comment by andreiglingeanu** [comment]
25a102a

**Comment by moldcraft** [comment]
Please test again https://github.com/ThemeFuse/Unyson-Extensions-Approval/issues/232
